SwiftOverlays/SwiftOverlays.swift

```diff
@@ -89,6 +89,22 @@ public extension UIViewController {
     func showImageAndTextOverlay(_ image: UIImage, text: String) -> UIView  {
         return SwiftOverlays.showImageAndTextOverlay(self.view, image: image, text: text)
     }
+
+   /**
```

Comment:
Is a Xcode convention for Documentation comments, please check this: http://nshipster.com/swift-documentation/